adm_program/system/classes/ModuleGroupsRoles.php

```diff
@@ -376,7 +376,7 @@ public function createContentPermissionsList()
         }
 
         // initialize and set the parameter for DataTables
-        $dataTables = new HtmlDataTables($this, 'role-permissions-table');
+        $dataTables = new HtmlDataTables($this, 'role_permissions_table');
```

Comment:
Could you add the prefix `adm_` to the ID so we have the naming convention of Admidio 5